db/migrate/20230731135634_create_foods.rb

```diff
@@ -0,0 +1,13 @@
+class CreateFoods < ActiveRecord::Migration[7.0]
+  def change
+    create_table :foods do |t|
+      t.string :name
+      t.string :measurement_unit
+      t.decimal :price, precision: 5, scale: 2 
+      t.integer :quantity
```

Comment:
:red_circle: [REQUIRED] I noticed that you have a column called quantity. However, this column is not used anywhere and is not part of the listed columns for this table. Unused columns are a bad practice when creating databases. They can take up unnecessary space in the database, and they can also make it more difficult to maintain the database. If you're not using a column, you should remove it.

Hi [student name],

I'm reviewing your ERD for the food table. I noticed that you have a column called quantity. However, this column is not used anywhere in the diagram.

Unused columns are a bad practice when creating databases. They can take up unnecessary space in the database, and they can also make it more difficult to maintain the database. If you're not using a column, you should remove it.

To remove the quantity column, you can follow these steps:
```

class RemoveQuantityColumnFromFoods < ActiveRecord::Migration[7.0]
  def change
    remove_column :foods, :quantity
  end
end

```

Reply:
@bienvenuushindi 
![recipe_erd_2_members](https://github.com/newhoteng/Recipe-app/assets/106236756/03760b1a-5cf7-468a-8f23-c5b0aa2d1f24)

- I don't think I have to remove this column from the foods table as it is present in the ERD as shown in the diagram above.

<img width="1055" alt="Screenshot 2023-08-03 at 1 05 46 PM" src="https://github.com/newhoteng/Recipe-app/assets/106236756/d5075d8f-4513-4f61-96a3-b13e44b7cc54">

- Secondly, Even though that column is not visible in the wireframe, the requirements asked for its inclusion as shown in the diagram above.